### PR TITLE
ssm parameters

### PR DESCRIPTION
**Pull request by lgothelipe**
db host, db name, db user and db password are getting from SSM.

### CODE REVIEW

**Comment by christina-roperto** [comment]
Hey Lucas, sorry I couldn't finish testing it because of some errors. Please see our slack channel for the error message.

File: Makefile [hunk 1]
```diff
@@ -3,11 +3,19 @@ COMPOSE_RUN = docker-compose run --rm tf012aws2
...
 #.SILENT
 
-.PHONY: plan build deploy shell
+.PHONY: prepare plan build deploy clean cmurlshell
 
-all: plan build deploy shell
+all: plan build deploy
+
+prepare:
+	cp .env.example .env
```
Comment (christina-roperto) [inline Makefile:11]: I'm getting ```Error: Error creating SNS topic: InvalidParameter: Invalid SMS endpoint: +61xxxxxxxxx
	status code: 400, request id: 7ed6fce3-26f9-56bf-a8f6-abfd7e69cba5``` despite updating my phone number on .env

Comment (lgothelipe) [reply Makefile:11]: edit the variables on .env.example only. Make plan will copy the .env.example to .env
I put this instructions on the README.md

1. Clone
2. Set up AWS credentials (aws configure)
3. Edit .env.example
4. make plan
5. make build
6. make deploy

or after 3. -> 4. make all

**Comment by christina-roperto** [comment]
I got `Error establishing a database connection` when accessing the public IP

**Review by christina-roperto — APPROVED** [review]
It's approved based on our convo on slack

**Pull request merged**